swagger/v1/swagger.json

```diff
@@ -0,0 +1,483 @@
+{
```

Comment:
This is what we're really after: an OpenAPI compatible document! This can be feed into documentation generation tools to create out API docs website